cmd/scheduler/app/options/options.go

```diff
@@ -134,6 +137,7 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 	fs.StringSliceVar(&s.NodeSelector, "node-selector", nil, "volcano only work with the labeled node, like: --node-selector=volcano.sh/role:train --node-selector=volcano.sh/role:serving")
 	fs.BoolVar(&s.EnableCacheDumper, "cache-dumper", true, "Enable the cache dumper, it's true by default")
 	fs.Uint32Var(&s.NodeWorkerThreads, "node-worker-threads", defaultNodeWorkers, "The number of threads syncing node operations.")
+	fs.IntVar(&s.WorkerNum, "worker-num", defaultWorkerNum, "The number of concurrent workers that can be used when multitasking is involved in the scheduler")
```

Comment:
multitasking? I think previous description is better, here we should explain which scheduling behaviors does this parameter take effect on specifiltitasking? I think previous description is better, here we should explain which scheduling behaviors does this parameter take effect on specifically, and `WorkerNum` param option should add the comments.

Reply:
Hi, plz have a feedback on this and sign off your commit: )